fuel_test/clear_env.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -eu
+
+. localrc
+
+[ $DEBUG == "on" ] && set -x
+
+function clear_xs {
+	local xs_host="$1"
+	ssh -qo StrictHostKeyChecking=no root@$xs_host \
```

Comment:
For FM, we can depend on snapshot to rollback. But should the slave nodes be deleted at here?